docs/log-data-in-reportportal/test-framework-integration/Python/pytest.md

```diff
@@ -9,6 +9,200 @@ There is an agent to integrate [pytest](https://docs.pytest.org/en/7.3.x/) with
+- `rp_launch_timeout = 86400` - Maximum time to wait for child processes finish, default value: 86400 seconds (1 day).
+- `rp_launch_uuid_print = True` - Enables printing Launch UUID on test run start. Default `False`.
+- `rp_launch_uuid_print_output = stderr` - Launch UUID print output. Default `stdout`. Possible values: [stderr, stdout].
+- `rp_parent_item_id = xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx` - id of the existing test item for session to use as parent item for the tests (the session will not handle the lifecycle of the given test item).
+- `rp_tests_attributes = 'PyTest' 'Smoke'` - list of attributes that will be added for each item in the launch.
+- `rp_connect_timeout = 15` - Connection timeout to ReportPortal server. Default value is "10.0".
+- `rp_read_timeout = 15` - Response read timeout for ReportPortal connection. Default value is "10.0".
+- `rp_log_batch_size = 20` - size of batch log request.
+- `rp_log_batch_payload_size = 65000000` - maximum payload size in bytes of async batch log requests.
+- `rp_log_level = INFO` - The log level that will be reported.
+- `rp_log_format = [%(levelname)7s] (%(name)s) %(message)s (%(filename)s:%(lineno)s)` - Format string to be used for logs sent to the service.
+- `rp_ignore_attributes = 'xfail' 'usefixture'` - Ignore specified pytest markers.
+- `rp_is_skipped_an_issue = False` - Treat skipped tests as required investigation. Default `True`.
+- `rp_hierarchy_dirs_level = 0` - Directory starting hierarchy level (from pytest.ini level) (default `0`).
+- `rp_hierarchy_dirs = True` - Enables hierarchy for tests directories, default `False`. Doesn't support 'xdist' plugin.
+- `rp_hierarchy_dir_path_separator` - Path separator to display directories in test hierarchy. In case of empty value current system path separator will be used (os.path.sep).
+- `rp_hierarchy_code` - Enables hierarchy for inner classes and parametrized tests, default `False`. Doesn't support 'xdist' plugin.
+- `rp_issue_system_url = https://bugzilla.some.com/show_bug.cgi?id={issue_id}` - issue URL (issue_id will be filled by parameter from pytest mark).
+- `rp_issue_id_marks = True` - Enables adding marks for issue ids (e.g. "issue:123456").
+- `rp_verify_ssl = True` - Verify SSL when connecting to the server.
+- `rp_mode = DEFAULT` - DEBUG or DEFAULT launch mode. DEBUG launches are displayed in a separate tab and not visible to anyone except owner.
+- `rp_thread_logging` - EXPERIMENTAL - Enables support for reporting logs from threads by patching the builtin Thread class. Use with caution.
+- `rp_api_retries = 0` - Amount of retries for performing REST calls to RP server.
+
+If you like to override the above parameters from command line, or from CI environment based on your build, then pass `-o "rp_launch_attributes=Smoke Tests"` during invocation.
+
+## Examples
+
+For logging of the test item flow to ReportPortal, please, use the python
+logging handler provided by plugin like bellow:
+
+in conftest.py:
+
+```
```

Comment:
I suggest to add `python` word here and in other similar cases as well just be sure that it will be formatted correctly as python code.